### DISCUSSION TITLE
How to start over backup failed over ssh (Detected an active backup operation for running domain.)

### DISCUSSION THREAD

**Discussion by Ark74** [abbbi/virtnbdbackup#120] [Q&A / Support] [answered]
Hello!
I was doing a backup over ssh, but the connection failed, didn't used screen :(
Once I was able to get the connection now every time I get.

```
2023-07-25 12:23:32] INFO root virtnbdbackup - main [MainThread]: Backup mode auto, target folder is empty: executing full backup.
[2023-07-25 12:23:32] INFO root virtnbdbackup - main [MainThread]: Libvirt library version: [8000000]
[2023-07-25 12:23:32] ERROR root virtnbdbackup - main [MainThread]: Detected an active backup operation for running domain.
[2023-07-25 12:23:32] ERROR root virtnbdbackup - main [MainThread]: Check with [virsh domjobinfo W10Test]
```

Checking with the suggestion,
```
$ virsh domjobinfo W10Test
Job type:         Unbounded   
Operation:        Backup      
Time elapsed:     43677510     ms
Temporary disk space use: 6.922 GiB
Temporary disk space total: 45.000 GiB
```

Bad news is that I deleted the previous folder to start over, but how can I delete de unbounded job?
Thanks in advance.

**Comment by abbbi** [comment, marked as answer]
You can use option -k to forcibly kill any running active block job for the domain:

**-k, --killonly        Kill any running block job**

the backup most probably failed in a way it wasnt able to tell libvirtd to stop the backup job already started.

**Reply by Ark74** [reply]
Oh!, what a shame :cold_sweat: it was that easy.
Now I'm using screen in case something weird happens, thank you very much for you help.

I'm kind of new to this fine tool.

**Reply by abbbi** [reply]
i just added a small commit so future versions make it easier to spot the -k option:

https://github.com/abbbi/virtnbdbackup/commit/3f3d8db7ede6f896d146ba087f967783d4347035

**Reply by Ark74** [reply]
Thank you very much.
You can mark this as solved when ever you want.
